tests/test_calls_recording.py

```diff
@@ -0,0 +1,44 @@
+
+import pytest
+import requests
+
+
+@pytest.fixture
+def endpoint_url(httpserver):
+    httpserver.expect_request("/foobar").respond_with_json({"foo": "bar"})
+    return httpserver.url_for("/foobar")
+
+
+@pytest.fixture
+def handler(httpserver, endpoint_url):
```

Comment:
handler should be better to returned from the "respond_with_json " and the other function as seen in my other PR
by this, there's no need to index the httpserver.handlers list